website/docs/guides/mesh-qs.md

```diff
@@ -0,0 +1,585 @@
+- Set dbt [version](/docs/dbt-versions/core) to 1.6 or later in your development and deployment [environments](/docs/dbt-cloud-environments) or select [Keep on latest version of](/docs/dbt-versions/upgrade-dbt-version-in-cloud#keep-on-latest-version) dbt. 
+- Load the Jaffle Shop sample data (customers, orders, and payments tables) into your data platform. You can find the steps to do so for each platform, respectively, here:
+  - [Snowflake](https://docs.getdbt.com/guides/snowflake?step=3)
+  - [Databricks](https://docs.getdbt.com/guides/databricks?step=3)
+  - [Redshift](https://docs.getdbt.com/guides/redshift?step=3)
+  - [BigQuery](https://docs.getdbt.com/guides/bigquery?step=3)
+  - [Fabric](https://docs.getdbt.com/guides/microsoft-fabric?step=2)
+  - [Starburst Galaxy](https://docs.getdbt.com/guides/starburst-galaxy?step=2)
+
+This guide assumes you have experience with or fundamental knowledge of dbt. Take the [dbt Fundamentals](https://courses.getdbt.com/courses/fundamentals) course first if you are brand new to dbt.
+
+## Create and configure two projects
+
+In this section, create two new, empty projects in dbt Cloud to use as your foundational and downstream projects. The always-enterprising and fictional account "Jaffle Labs" is setting up a project for their data analytics and finance team, respectively:
+
+<Lightbox src="/img/guides/dbt-mesh/project_names.png" width="50%" title="Create two new dbt Cloud projects named 'Jaffle | Data Analytics' and 'Jaffle Finance' " />
+
+To [create](/docs/cloud/about-cloud-setup) a new project in dbt Cloud:
+
+- From **Account settings**, click **+ New Project**.
+- Enter a project name and click **Continue**.
+  - Use "Jaffle | Data Analytics" for one project
+  - Use "Jaffle | Finance" for the other project
+- Select your data platform, then **Next** to set up your connection.
+- In the **Configure your environment** section, enter the **Settings** for your new project.
+- Click **Test Connection**. This verifies that dbt Cloud can access your data platform account.
+- Click **Next** if the test succeeded. If it failed, you might need to go back and double check your settings.
+- For this guide, make sure you create a single [development](/docs/dbt-cloud-environments#create-a-development-environment) and [Deployment](/docs/deploy/deploy-environments) per project.
+  - For "Jaffle | Data Analytics", set the default database to `jaffle_da`.
+  - For "Jaffle | Finance", set the default database to `jaffle_finance`
+
+<Lightbox src="/img/guides/dbt-mesh/create-new-project.gif" width="80%" title="Navigate to 'Account settings' and then click + 'New Project' to create new projects in dbt Cloud" /> 
+
+Once configured, each project should have:
+  - A data platform connection
+  - New git repo
+  - one or more [environments](/docs/deploy/deploy-environments) (such as development, deployment)
+
+## Set up a foundational project
+
+A foundation project is where 
+
```

Comment:
@adinsmoor can i confirm if this is right?

Reply:
I'll tweak this to upstream project